build/charts/antrea/templates/webhooks/validating/crdvalidator.yaml

```diff
@@ -148,7 +148,7 @@ webhooks:
     rules:
       - operations: ["CREATE", "UPDATE", "DELETE"]
         apiGroups: ["crd.antrea.io"]
-        apiVersions: ["v1alpha2"]
+        apiVersions: ["v1beta1"]
```

Comment:
we should keep validating the old version (even if it is deprecated) until we actually remove it?
I think this is what we usually do, but I may be wrong

Reply:
My understanding is that we will always convert the old version to the latest version by using conversion webhook. In this way, we only need to validate the new version.

Reply:
I think mutating webhooks are called before validating webhooks, but not sure about conversion webhook, could you double check?

Reply:
I tested this on my local setup previously and conversion webhook is always called before validation webhook.

Reply:
FYI: I found a chapter [here](https://www.oreilly.com/library/view/programming-kubernetes/9781492047094/ch04.html) about the validation step in the handler stack of apiextensions-apiserver.
According to the sequence, the request conversion is done before mutating and validating webhooks.
![image](https://github.com/antrea-io/antrea/assets/80799348/0de54c3e-dd4c-4a43-b1ba-92f1eb26ba75)